modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextBar.vue

```diff
@@ -512,7 +512,7 @@ export default {
 
       if (refreshable) {
         refreshable.innerHTML = content;
-        if (!this.original) {
+        if (this.editMode && (!this.original)) {
```

Comment:
At some point we started using this refresh method even in cases where we're not in edit mode (or not yet), which means we were grabbing an "original" that might lack content, resulting in an improper basis for the patches from undo, and... scene.